wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -13,7 +13,9 @@ end
 
 namespace :wagn do
   desc "create a wagn database from scratch"
-  task :create do
+  task :recreate do
```

Comment:
fwiw, I never used the term "deck app" for anything.

A Decko installation is a *deck*.  I suggested you can have a *rails deck* or an *app deck*.  Perhaps it would be less confusing to say *engine deck* vs *app deck*?  And for now you mentioned using *free* deck to mean something that is neither. So we could add:
````
#app_deck? 
#engine_deck?
#free_deck?
````
We know that technically, a rails app is also an engine, but an app deck is not an engine deck, ok?

If you're working with an *engine deck*, you should be installing decko-rails but the wagn/decko gem should not even be present, correct?

None of this answers my question above.  How the heck is this currently passing semaphore??